### DISCUSSION TITLE
No RPM package for Fedora

### DISCUSSION THREAD

**Discussion by ffimnsr** [citusdata/citus#7157] [General]
I've checked the package repository for RPM's and it only contains RPM's for CentOS and OracleLinux distribution. There's package for older Fedora versions up to 28 and not using the latest citus version. Anyway's why is that? 

Is it because there's no maintainer. I can handle the maintaining of that package, if ever.
I have created a copr build for it you can check it out here: https://copr.fedorainfracloud.org/coprs/eabucay/postgresql-extensions/.
And here is the repo for the spec file: https://github.com/ffimnsr/fedora-packager--citus/blob/main/citus.spec. I'll see if this can go through the main Fedora package management.

**Comment by hanefi** [comment]
In the past, we had Fedora support. IIRC we had a hard time getting some of the dependencies to build this project and that's why we dropped support.

It would be nice to have support again for Fedora. We only support rpm packages for el7, el8, ol7, ol8 right now. You can see our spec file at https://github.com/citusdata/packaging/blob/all-citus/citus.spec

**Reply by hanefi** [reply]
If you decide to open a PR on packaging repo that adds Fedora support, I'd be more than happy to help with setting up CI, helping you create docker images to create packages on Fedora etc.
